### PR TITLE
Add ability to run plugin manager as modal window

### LINKED ISSUE #13: Plugin manager as modal window over GitExtensions
As suggested in #4 and #5, make PM a modal window.

### PR DESCRIPTION
**Pull request by ivangrek**
fixes #13

### CODE REVIEW

File: src/GitExtensions.PluginManager/Plugin.cs [hunk 1]
```diff
@@ -64,7 +61,10 @@ public override bool Execute(GitUIEventArgs gitUiCommands)
                 Arguments = args.ToString(),
                 UseShellExecute = false,
             };
-            Process.Start(info);
+
+            var process = Process.Start(info);
```
Comment (RussKie) [inline src/GitExtensions.PluginManager/Plugin.cs:65]: In this case we'd want to dispose the process
```suggestion
            using var process = Process.Start(info);

```

**Comment by RussKie** [comment]
Can you please provide more information what this change does. It is not clear what the original issue is, and what we are gaining with the change.

**Comment by ivangrek** [comment]
After we run plugin manager it works like modal window and prevent run another PM window.

**Comment by ivangrek** [comment]
For test. I am building a UI project and changing the file along the path.

image

**Comment by ivangrek** [comment]
"C:\\Users\\USERNAME\\AppData\\Local\\GitExtensions\\UserPlugins"

**Comment by maraf** [comment]
I think this doesn't prevent launching plugin manager from other instance of Git Extensions.
Maybe we can use [WindowsFormsApplicationBase](https://dzimchuk.net/single-instance-of-a-wpf-app-e28093-part-4--windowsformsapplicationbase-/) to allow only a single instance of plugin manager.

**Comment by ivangrek** [comment]
CloseAllOtherInstances?

image

**Comment by maraf** [comment]
The snippet you picked closes all running instances of Git Extensions. The issue referes to instances of Plugin Manager.

**Review by maraf — APPROVED** [review]
Good job 👍
Thank you.

**Comment by RussKie** [comment]
Interop definitions can be improved, but it isn't critical.
LGTM

**Comment by mast-eu** [comment]
Thank you.

**Pull request merged**